### PR TITLE
Omtfdev mk18

### PR DESCRIPTION
**Pull request by konec**
OMTF:
Fill RegionalMuonCandBxCollection with order assumed by GMT (i.e. sorted by GMT links).
Modify BXVector.impl since insert does not work.

### CODE REVIEW

File: EventFilter/L1TRawToDigi/src/OmtfMuonUnpacker.cc [hunk 1]
```diff
@@ -27,8 +27,20 @@ void MuonUnpacker::unpack(unsigned int fed, unsigned int amc, const MuonDataWord
   digi.setTrackAddress(trackAddr);
   digi.setTFIdentifiers(iProcessor, overlap);
   int bx = data.bxNum()-3;
-  LogTrace("")  <<"OMTF Muon, BX="<<bx<<", hwPt="<<digi.hwPt()<< std::endl;
-  if(std::abs(bx) <= 3) producedMuonDigis->push_back(bx,digi);
+  LogTrace("")  <<"OMTF Muon, BX="<<bx<<", hwPt="<<digi.hwPt()<<", link: "<<digi.link() << std::endl;
+
+  // add digi to collection, keep fixed ascending link orderi (insert in proper place)
+  if(std::abs(bx) <= 3) {
```
Comment (thomreis) [inline EventFilter/L1TRawToDigi/src/OmtfMuonUnpacker.cc]: Why only for BX range +/-2? Should it not work the same for all BX?

Comment (konec) [reply EventFilter/L1TRawToDigi/src/OmtfMuonUnpacker.cc]: Hi, I have not changed BX range. It is/was [-3,+3].  OMTF in maximum (it depends on settings) stores 8BXes with "3rd" as the trigger one.

Comment (thomreis) [reply EventFilter/L1TRawToDigi/src/OmtfMuonUnpacker.cc]: Sorry, I meant +/-3 of course. So the maximum of OMTF would be [-3,+4] then? Would this work if the "if" is just removed?

Comment (konec) [reply EventFilter/L1TRawToDigi/src/OmtfMuonUnpacker.cc]: There are 3 bits for BX in data.

**Comment by konec** [comment]
Hi @thomreis, @kbunkow 
After talk to Karol, I change -3,3 to -3,4.

**Pull request merged**